docs/api/networkinterfaces/NetworkStatus.md

```diff
@@ -2,7 +2,7 @@
 
 This interface informs you about connection state changes asynchronously. Providing a method to register a callback function to a socket accomplishes this. Each time the network interface's state changes, it triggers the callback.
 
-<span class="images">![](https://s3-us-west-2.amazonaws.com/mbed-os-docs-images/NetworkinterfaceStates.png)<span>Network states</span></span>
+<span class="images">![](../../images/NetworkinterfaceStates.png)<span>Network states</span></span>
```

Comment:
I don't know if this format will render nicely, but I'm interested to try it and see.